Add `Beatmap/Pattern/osu!mania` listing and subpages (#8595)
ppy/osu-wiki: wiki/Beatmap/Pattern/Mania/Chordstreams/en.md

---
stub: true

> tags:
>   - chordstreams

Reviewer: ```suggestion
```

---

> # Chordstreams

Reviewer: ```suggestion
# Chordstream
```

_For stream types specific to 4-key, refer to [streams](mania/streams)._

> **Chordstreams** are a type of [stream](wiki/pattern/mania/streams/en.md#streams) which involve different chord sizes. The stream is usually on 1/4th with chords every 1/1st or 1/2nd snap interval. In keymodes outside of 4-key osu!mania, **chordstream** usually refers to streams involving 3 or more notes within the chords.

Reviewer 2: I would maybe clarify here that people call "Light" chordstreams patterns that use chords on 1/1st or maybe 1/2nds whereas "Dense" chordstreams usually use 1/4th for chords too instead than double streams. I personally have never heard double streams before but I am not too big of a 7k guy anyway.

> **Chordstreams** are a type of [stream](wiki/pattern/mania/streams/en.md#streams) which involve different chord sizes. The stream is usually on 1/4th with chords every 1/1st or 1/2nd snap interval. In keymodes outside of 4-key osu!mania, **chordstream** usually refers to streams involving 3 or more notes within the chords.

Reviewer: ```suggestion
**Chordstreams** are a type of [stream](wiki/pattern/mania/streams/en.md#streams) which involve different chord sizes. The stream is usually on 1/4 with chords every 1/1 or 1/2 snap interval. In key modes outside of 4K, "chordstream" usually refers to streams involving 3 or more notes within the chords.
```

![](img/chordstream.png)

> **Chordstream** may also include **double streams**, which are two streams occurring simultaneously. These are more common in 7-key osu!mania and key modes above.

Reviewer: ```suggestion
**Chordstreams** may also include **double streams**, which are two streams occurring simultaneously. These are more common in 7K osu!mania or higher key modes.
```

![](img/chordstream2.png)

> ## Brackets

Reviewer: ```suggestion
## Bracket
```

> A specific type of chordstream involves the usage of 2 or more [trills](wiki/pattern/mania/trills/en.md#trills) occurring at the same time are known as **brackets**, though the trills may switch columns. They usually occur when chordstreams are quite dense with its chord usage and usually mix both one-handed trills and two-handed trills together.

Reviewer: ```suggestion
**Brackets** are a specific type of chordstream involving the usage of 2 or more [trills](wiki/pattern/mania/trills/en.md#trills) occurring at the same time, though the trills may switch columns. They usually occur when chordstreams are quite dense with their chord usage and usually mix both one-handed and two-handed trills together.
```
